src/components/Filters/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
```

Comment:
🤦  😅

Reply:
Hm, what can we do. Now I believe there is not better option than `react-datepicker` 😄